diff --git a/muse3/CMakeLists.txt b/muse3/CMakeLists.txt
index 2746a50..63d3a5e 100644
--- a/muse3/CMakeLists.txt
+++ b/muse3/CMakeLists.txt
@@ -169,7 +169,7 @@ option ( ENABLE_LV2_MAKE_PATH "Enable LV2 makePath support. Stores plugin config
 option ( ENABLE_LV2_DEBUG    "Enable LV2 debugging messages" OFF)
 option ( ENABLE_FLUID        "Enable fluidsynth softsynth plugins."                                ON)
 option ( ENABLE_RUBBERBAND   "Enable RubberBand Audio Stretcher/Pitch Shifter/Sample Rate Converter" ON)
-#option ( ENABLE_ZITA_RESAMPLER "Enable Zita Resampler Audio Sample Rate Converter" ON)  # TODO See below...
+option ( ENABLE_ZITA_RESAMPLER "Enable Zita Resampler Audio Sample Rate Converter" ON)  # TODO See below...
 option ( ENABLE_MIDNAM       "Enable MIDI Name support (MidNam). Official MMA instrument spec. Also for LV2 plugins." ON)
 option ( ENABLE_INSTPATCH    "Enable Instrument Patch Library support (enhances soundfont support)." ON)
 option ( ENABLE_EXPERIMENTAL "Enable various experimental features (not recommended)."              OFF)
