diff --git a/muse3/awl/knob.cpp b/muse3/awl/knob.cpp
index be8bc85d..b9f1e3e2 100644
--- a/muse3/awl/knob.cpp
+++ b/muse3/awl/knob.cpp
@@ -25,6 +25,7 @@
 
 #include <QMouseEvent>
 #include <QPainter>
+#include <QPainterPath>
 
 namespace Awl {
 
diff --git a/muse3/awl/slider.cpp b/muse3/awl/slider.cpp
index 7d82fde3..e33be869 100644
--- a/muse3/awl/slider.cpp
+++ b/muse3/awl/slider.cpp
@@ -24,6 +24,7 @@
 
 #include <QMouseEvent>
 #include <QPainter>
+#include <QPainterPath>
 
 namespace Awl {
 
diff --git a/muse3/muse/components/slider.cpp b/muse3/muse/components/slider.cpp
index 8d27043b..400bf0cc 100644
--- a/muse3/muse/components/slider.cpp
+++ b/muse3/muse/components/slider.cpp
@@ -27,6 +27,7 @@
 
 #include <QPainter>
 #include <QResizeEvent>
+#include <QPainterPath>
 
 #include "utils.h"
 #include "slider.h"
diff --git a/muse3/muse/components/utils.cpp b/muse3/muse/components/utils.cpp
index 06600f03..02e97e3d 100644
--- a/muse3/muse/components/utils.cpp
+++ b/muse3/muse/components/utils.cpp
@@ -37,6 +37,7 @@
 #include <QPointF>
 #include <QFileInfo>
 #include <QFont>
+#include <QPainterPath>
 
 #include "audio.h"
 #include "audiodev.h"
diff --git a/muse3/muse/widgets/meter.cpp b/muse3/muse/widgets/meter.cpp
index 2daf6699..b3786899 100644
--- a/muse3/muse/widgets/meter.cpp
+++ b/muse3/muse/widgets/meter.cpp
@@ -31,6 +31,7 @@
 #include <QResizeEvent>
 #include <QVector>
 #include <QLocale>
+#include <QPainterPath>
 #include <algorithm>
 
 #include "meter.h"
