diff --git a/muse3/muse/components/midi_audio_control.cpp b/muse3/muse/components/midi_audio_control.cpp
index 4c29cd3..3b9c725 100644
--- a/muse3/muse/components/midi_audio_control.cpp
+++ b/muse3/muse/components/midi_audio_control.cpp
@@ -29,9 +29,11 @@
 #include "midictrl.h"
 #include "audio.h"
 #include "app.h"
+#include "operations.h"
 
 #include <QTimer>
 
+
 namespace MusEGui {
 
 // -----------------------------------
