diff -up muse-muse_3_1_0/muse3/muse/app.cpp.omv~ muse-muse_3_1_0/muse3/muse/app.cpp
--- muse-muse_3_1_0/muse3/muse/app.cpp.omv~	2020-04-02 05:46:31.881352829 +0200
+++ muse-muse_3_1_0/muse3/muse/app.cpp	2020-04-02 05:48:23.125719521 +0200
@@ -579,7 +579,7 @@ MusE::MusE() : QMainWindow()
       midiInputFilterAction = new QAction(QIcon(*MusEGui::midi_inputplugins_midi_input_filterIcon), tr("Midi Input Filter..."), this);
       midiRemoteAction = new QAction(QIcon(*MusEGui::midi_inputplugins_remote_controlIcon), tr("Midi Remote Control..."), this);
 #ifdef BUILD_EXPERIMENTAL
-      midiRhythmAction = new QAction(QIcon(*midi_inputplugins_random_rhythm_generatorIcon), tr("Rhythm Generator"), this);
+      midiRhythmAction = new QAction(QIcon(/* FIXME This doesn't have an icon yet... *MusEGui::midi_inputplugins_random_rhythm_generatorIcon */), tr("Rhythm Generator"), this);
 #endif
       midiResetInstAction = new QAction(QIcon(*MusEGui::midi_reset_instrIcon), tr("Reset Instrument"), this);
       midiInitInstActions = new QAction(QIcon(*MusEGui::midi_init_instrIcon), tr("Init Instrument"), this);
